tests/unit/plugins/test_ant_plugin.py

```diff
@@ -15,13 +15,12 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import pytest
-from pydantic import ValidationError
-
 from craft_parts import Part, PartInfo, ProjectInfo, errors
 from craft_parts.plugins.ant_plugin import AntPlugin
+from pydantic import ValidationError
```

Comment:
Same third-party vs local reordering here. Is isort happy with that? Otherwise it could be a ruff bug or configuration.

Reply:
isort moved it to the top